tests/providers/amazon/aws/hooks/test_rds.py

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import unittest
+
+import boto3
+
+from airflow.providers.amazon.aws.hooks.rds import RDSHook
+
+try:
+    from moto import mock_rds
+except ImportError:
+    mock_rds = None
+
+
+class TestRDSHook(unittest.TestCase):
```

Comment:
since you're not implementing methods anymore, just `conn` maybe it is unnecesssary to test these client methods -- since those are just boto features

maybe it would be better to simply test that `hook.conn` returns the right client (namely an RDS client)

e.g. i think this works

```python
hook = RDSHook()
conn = hook.conn
assert conn.__class__.__name__ == 'RDS'
```
in trying this out locally i had to patch env var with `AIRFLOW_CONN_AWS_DEFAULT=s3://@` but YMMV

i'm updating test util `env_vars` to allow this [here](https://github.com/apache/airflow/pull/20818).

then i think we  can  remove everything related to mock_rds